lib/src/ui/puzzle/puzzle_dashboard_widget.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:fl_chart/fl_chart.dart';
+import 'package:fast_immutable_collections/fast_immutable_collections.dart';
+
+import 'package:lichess_mobile/src/model/puzzle/puzzle_providers.dart';
+import 'package:lichess_mobile/src/widgets/list.dart';
+import 'package:lichess_mobile/src/widgets/platform_card.dart';
+import 'package:lichess_mobile/src/model/puzzle/puzzle.dart';
+import 'package:lichess_mobile/src/ui/puzzle/puzzle_dashboard_screen.dart';
+
+class PuzzleDashboardWidget extends ConsumerWidget {
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final days = ref.watch(daysProvider);
+    final puzzleDashboard =
+        ref.watch(puzzleDashboardProvider(days.toInteger()));
+
+    return puzzleDashboard.when(
+      data: (data) {
+        return ListSection(
+          header: const Text('Puzzle Dashboard'),
+          children: [
+            CustomPlatformCardRow([
+              CustomPlatformCard(
+                'Performance',
+                value: data.global.performance.toString(),
+              ),
+              CustomPlatformCard(
+                'Played',
+                value: data.global.nb.toString(),
+              ),
+              CustomPlatformCard(
+                'Solved',
+                value:
+                    '${((data.global.firstWins / data.global.nb) * 100).toInt()}%',
+              ),
+              CustomPlatformCard(
```

Comment:
Don't show this one for now, because the replay feature is not available to the app yet.